### ISSUE TITLE
DocumentBuilder return "urls": {} if `/var/lib/onlyoffice` mounted

### ISSUE THREAD

**Issue by ShockwaveNN** [ONLYOFFICE/Docker-DocumentServer#164]
**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
1. run document server with
`docker run -i -t -p 80:80 -v /app/onlyoffice/DocumentServer/lib:/var/lib/onlyoffice onlyoffice/documentserver:5.3.0.243`
2. Send request to DocumentBuilder to `http://localhost/docbuilder?async=false`
with body
```
builder.CreateFile("docx");
var oDocument = Api.GetDocument();
var oParagraph, oRun;
oParagraph = oDocument.GetElement(0);
oRun = Api.CreateRun();
oRun.AddText("This is just a sample text. Nothing special.");
oParagraph.AddElement(oRun);
builder.SaveFile("docx", "/tmp/add_text.js20180324-4106-83z5wy.docx");
builder.CloseFile();
```
3. Get
```
{
    "key": "bld_db7c917505909e2a",
    "urls": {},
    "end": true
}
```

If docker container run without mount `/var/lib/onlyoffice` we got file in responce

Seems problem that if `/var/lib/onlyoffice` not mounted there is `/var/lib/onlyoffice/documentserver/App_Data/docbuilder/` folder
If `/var/lib/onlyoffice` is mounted - do not exists `/var/lib/onlyoffice/documentserver/App_Data/docbuilder/`
**DocumentServer Docker tag:** 5.3.0.243

**Host Operating System:** Ubuntu 18.04

**Comment by agolybev** [comment]
I fix it by creation `/var/lib/onlyoffice/documentserver/App_Data/docbuilder` inside container but I can't verify solution because I always get response from documentserver like this: `{"key":"bld_9817bd66bd4ed698","urls":{},"end":true}`

**Comment by ShockwaveNN** [comment]
Can confirm fix on `onlyoffe/4testing-documentserver-ie:5.3.1.18`

**Comment by ShockwaveNN** [comment]
Fixed on `onlyoffice/documentserver:5.3.1.26`
